processmanager/types.go

```diff
@@ -148,6 +151,8 @@ type processInfo struct {
 	mappingsByFileID map[host.FileID]map[libpf.Address]*Mapping
 	// C-library Thread Specific Data information
 	tsdInfo *tpbase.TSDInfo
+	// process env vars from /proc/PID/environ
```

Comment:
Can you rebase your branch on top of `main` to include the changes from https://github.com/open-telemetry/opentelemetry-ebpf-profiler/pull/344 and move `envVariables map[string]string` into `ProcessMeta`?

Reply:
Done